public/index.html

```diff
@@ -15,23 +15,26 @@
 		<!-- Modals templates get rendered in here -->
 	</div>
 
+	<script>
+		const base_url = "{{ base_url }}";
```

Comment:
Ideally this would be in setup.js and we'd move that one to be loaded before other js files, but I can't seem to find a way to convince jinja to replace it in a module directly, so I guess it can stay for now.